site-templates/pages/overview.gomd

```diff
@@ -3,7 +3,7 @@
   <div class="col-md-8 content">
   
     <h1>Overview</h1>
-    <p>The Bluescape API is a <a href="/docs/page/reference">REST API</a> that can be accessed through explicit HTTP calls. These APIs expose most of the features available in the Bluescape product. The Bluescape API enables developers to programmatically create, upload, and update content within Bluescape workspaces. Applications are able to pull data and files from external sources like databases, content management systems and cloud storage and push them into Bluescape workspaces with ease. Organizations are free to move beyond the Bluescape web and wall interfaces for integrating the Bluescape visual layer into their collaboration and decision-making processes.</p>
+    <p>The Bluescape API is a <a href="/docs/page/reference">REST API</a> that can be accessed through explicit HTTP calls. These APIs expose most of the features available in the Bluescape product. The Bluescape API enables developers to programmatically create, upload, and update content within Bluescape workspaces. Applications are able to pull data and files from external sources like databases, content management systems and cloud storage and push them into Bluescape workspaces with ease. Organizations are free to move beyond the Bluescape web and Wall interfaces for integrating the Bluescape visual layer into their collaboration and decision-making processes. Bluescapes GraphQL service will be coming soon in order to further extend the APIs capabilities. This will not replace the REST APIs, and only servers to add additional tools for developers.</p>
```

Comment:
What does "only servers to add.." mean?

Reply:
typo, good catch!
Supposed to say SERVES.
As in "this serves to add additional features"

Reply:
And hey... i'm going for positive reinforcement. If you believe it's the last, then maybe it will be! :)